ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -856,6 +856,19 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                               lrn->get_nsize());
             break;
         }
+        case OP_TYPEID::MatMul:
```

Comment:
As I understand all method which have `evaluate` are calculated during Interpreter execution (look here: https://github.com/mbencer/openvino/blob/926be8356887567e1f58a7e55bc34a01cc660ed1/ngraph/test/runtime/interpreter/int_executable.cpp#L205). Why is called directly?